packages/firestore/test/integration/api_internal/idle_timeout.test.ts

```diff
@@ -25,7 +26,9 @@ apiDescribe('Idle Timeout', persistence => {
       return docRef
         .set({ foo: 'bar' })
         .then(() => {
-          return drainAsyncQueue(db);
+          return asyncQueue(db).runDelayedOperationsEarly(
+            TimerId.WriteStreamIdle
```

Comment:
This would also read nicer to me if this was a constant on WriteStream.

Reply:
I was going to suggest a String but I think you already changed my mind of that.